tools/utils/Utils/Properties/AssemblyInfo.cs

```diff
@@ -11,11 +11,11 @@
 // General Information about an assembly is controlled through the following 
 // set of attributes. Change these attribute values to modify the information
 // associated with an assembly.
-[assembly: AssemblyTitle("Microsoft.Packaging.Utils.Properties")]
+[assembly: AssemblyTitle("Microsoft.Msix.Utils.Properties")]
 [assembly: AssemblyDescription("Library for utilities like AppxPackaging, AppxPackagingInterop, CommandLine, IO, Logger, ProcessRunner, and StandardInput.")]

```

Comment:
Does anybody have any suggestion for the package description? I used what we had in the readme as a placeholder, but it's probably better to change it. Something like "Library containing utility types for MSIX, and interop for the Packaging API", perhaps?

Reply:
your proposal sounds reasonable

---
In reply to: [468280667](https://github.com/microsoft/msix-packaging/pull/369#discussion_r468280667) [](ancestors = 468280667)